tests/ert/unit_tests/forward_model_runner/test_forward_model_step.py

```diff
@@ -295,3 +295,104 @@ def test_processtree_timer(
     for snapshot in snapshots:
         timer.update(snapshot)
     assert timer.total_cpu_seconds() == expected_total_seconds
+
+
+@pytest.mark.usefixtures("use_tmpdir")
+@pytest.mark.parametrize(
+    "exit_code, expected_error_message",
+    [(0, None), (1, "Process exited with status code 1")],
+)
+def test_target_file_specified(exit_code, expected_error_message):
```

Comment:
Can this test name be more explicit about what it is testing? 
Are we testing propagation of error messages, or how Ert handles the step when the target file is specified and is present, or both?

Are we testing that Ert is picking up the correct exit code (which it might be able to do, even when the exit code is to be ignored due to the target_file) - we should probably verify all these corner cases.

Reply:
Maybe I would like to see a parameter to the test which tells me if Ert regard the step execution as a success or not.